blog_api/lib/blog_api/post/list.ex

```diff
@@ -0,0 +1,13 @@
+defmodule BlogApi.Post.List do
```

Comment:
@niina1 Geralmente o moduledoc fica junto com o defmodule:
```suggestion
defmodule BlogApi.Post.List do
 @moduledoc """
  Busca todos Posts no banco de dados
  """

```